colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb

```diff
@@ -0,0 +1,765 @@
+{
```

Comment:
<p>I would add some configs here too, like batch size, model name etc etc unless these get picked up by the logger itself?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
good catch, it appears they don't get logged